content/contributing/core-team/contents.lr

```diff
@@ -0,0 +1,86 @@
+---
+body:
+
+Those busy bees on the `Core Team </community/team>`_ have a number of responsibilities to keep the hive that is BeeWare moving. This is an evolving project, so this page is subject to change.
+
+These include, but aren't limited to, responding to issues, reviewing and merging code, and architecture of the BeeWare project as a whole. 
+
+
+There are people we trust to make code decisions, there are people we trust to make code and organizational decisions, and there is a person with whom the metaphorical buck stops, who guides the vision of the whole organization and is often it's biggest cheerleader
+
+These levels can be described as follows: 
+
+Bee, or Worker Bee:
+ - Any member of the BeeWare community. Given we work in the open on GitHub, anyone can suggest changes to the code and have their code merged. The only limit to your ability to contribute is having your work merged by a team member with the rights to do so. 
+
+Apiarist:
+ - Any Bee with a Commit Bit. These members have demonstrated ability in relation to the project -- this could be on a technical level (JavaScript, Python, Objective-C expertise; GTK, macOS knowledge), or on another level (community management, code review). 
+
+Senior Apiarists: 
+ - Apiarists with elevated access in GitHub, and also an added level of responsibility to oversee the project as a whole. They are able to make architectural decisions, but ultimately answer to the BDFN
+
+Founding Apiarist: `Russell Keith-Magee <https://twitter.com/freakboy3742>`_
+ - The man that first stood on a hill and spied a yak that needed shaving.  
+ - This role never changes, and continues ad infinitum.
+ - This role is different to the BDFN role. 
+
+Bee-nevolent Dictator for Now (BDFN): `Russell Keith-Magee <https://twitter.com/freakboy3742>`_
+ - A take on `Benevolent Dictator for Life <https://en.wikipedia.org/wiki/Benevolent_dictator_for_life>`_, the buck stops with the BFDN. The use of "For Now" as opposed to "For Life" is reference to the Django theme of not subjecting the responsibilities of core maintainer for a person's entire natural life. Life exists outside open source, and code/life balance and general well-being is a very important thing to keep in mind. 
+
+
+Guidelines (not actual rules)
+------------------------------------
+
+As with any project with more than one person with commit rights, there are a number of general guidelines that the team should follow: 
+
+
+ * **Be a good representation of the project to the wider community**
+
+ * **Treat every enquiry and contribution to any BeeWare project with respect.**
+
+ * **Assume everyone has good intentions, even if they haven't chosen their words well.**
+
+ * **Assume that if someone has done something the "wrong" way, it's because we've failed in communicating process.**
+ 
+ * **Assume any expression of anger or frustration comes from a genuine place of wanting to use a BeeWare tool/library.**
```

Comment:
Add:  * Encourage other members of the community to reflect these ideals in their own communications, both inside and outside the BeeWare community.

The idea here: One of the biggest contributions Jacob and Adrian made to Django was clamping down on bad language/bad behaviour in Django fora. This "leading by example" is part of what made the Django community such a pleasant place. It also set the scene for CoC discussions a few years later, because it's a lot easier to have a conversation about civility when everyone is being civil (c.f., Linus.)